src/test/jest/results/59c86272-03ea-42e9-a62d-1c8a272e8ab0.json

```diff
@@ -0,0 +1,112 @@
+[
+    {
+        "features": {
+            "aggregates": {
+                "PC_0": {
+                    "dtlsErrors": 0,
+                    "dtlsFailure": 0,
+                    "iceFailed": false,
+                    "iceReconnects": 0,
+                    "inboundVideoExperience": {
+                        "lowerBoundAggregates": {
+                            "meanFrameHeight": 630,
+                            "meanFramesPerSecond": 29.75
+                        },
+                        "upperBoundAggregates": {
+                            "meanFrameHeight": 630,
+                            "meanFramesPerSecond": 29.75
+                        }
+                    },
+                    "isP2P": true,
+                    "pcSessionDurationMs": 41170,
+                    "trackAggregates": {
+                        "receivedPacketsLostPct": 0,
+                        "sentPacketsLostPct": 0,
+                        "totalPacketsReceived": 4551,
+                        "totalPacketsSent": 0,
+                        "totalReceivedPacketsLost": 0,
+                        "totalSentPacketsLost": 0
+                    },
+                    "tracks": {
+                        "receiverTracks": [
+                            {
+                                "mediaType": "video",
+                                "packets": 4551,
+                                "packetsLost": 0,
+                                "packetsLostPct": 0,
+                                "packetsLostVariance": 0,
+                                "ssrc": "3155040392",
+                                "videoType": "camera"
```

Comment:
Great that you added a new test case for this already!

How hard would it be to extend this test case to actually switch the `videoType` on the same SSRC?
And/or maybe add a second track with another `videoType`?

Reply:
Thanks @nils-ohlmeier, that's a good point I'll make sure to add such a test case.